graql/language/rule-validation.feature

```diff
@@ -305,6 +305,36 @@ Feature: Graql Rule Validation
       """
 
 
+  Scenario: when a rule creates long attribute for a double attribute type, no error is thrown
```

Comment:
I actually prefer a more positive description - at first glance, my eyes assumed it said "an error is thrown" and I had to look again a few times:
```suggestion
  Scenario: a rule can create a long attribute for a double attribute type
```

Reply:
updated a bunch